.gitignore

```diff
@@ -1,46 +1,523 @@
-.DS_Store
-.directory
-.idea
-.kdev4
-__pycache__
-*~
-build
-*.mo
-dist
+# moban hashes
+.moban.hashes
+
+# Extra rules from https://github.com/github/gitignore/
+# Python rules
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+*.egg-info/
+.installed.cfg
+*.egg
 MANIFEST
-*.pyc
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit test / coverage reports
+htmlcov/
+.tox/
 .coverage
-*.patch
-_build
-coverage.xml
-htmlcov
-*.orig
-*.diff
-*.kdev4
-*.patch
-*.sublime-project
-*.sublime-workspace
-.python-version
-coala.egg-info
 .coverage.*
-coala.1
-src
-site
-.zanata-cache
-*.exe
 .cache
-*.eggs
-node_modules
+nosetests.xml
+coverage.xml
+*.cover
+.hypothesis/
+.pytest_cache/
+
+# Translations
+*.mo
+*.pot
+
+# Django stuff:
 *.log
-*.jar
```

Comment:
This comes from https://github.com/coala/coala/issues/1412 .

All that has moved to coala-bears, and only some docs in this repo refer .jar